src/RoleObjects/ButtonData.js

```diff
@@ -8,17 +8,6 @@ export default class ButtonData extends AccessibilityObject {
     this._reactProps.onClick = this._onClick;
   }
 
-  _onClick(evt) {
```

Comment:
Just moving this down so that public functions are listed before protected/private ones.